### ISSUE TITLE
Respect monthly traffic limit given by accounting server

### ISSUE THREAD

**Issue by audax** [Qabel/qabel-block#30]
Requires: https://github.com/Qabel/qabel-accounting/issues/92

Extract the download traffic column to another table with the columns user_id, traffic and month. The current download_traffic should be accounting for may.

The maximum download traffic per month is given by the accounting server in the response of an auth request and should be cached, but not saved.

**Comment by enkore** [comment]
Naturally this only affects authenticated users. GETs can be made by anyone, even without authorization. Limiting those *per se* doesn't make sense (maybe per IP? --> separate issue).
